<launch>
  <param name='/kinect2/qhd/image_ir_rect/compressed/jpeg_quality' value='15'/>
  <param name="/amcl/update_min_d" value="0.02"/>
  <param name="/amcl/update_min_a" value="0.05"/>

  <node pkg="Tus_g5" type="main_ctrl" name="main_ctrl" output="screen"/>
  <node pkg="Tus_g5" type="map_ctrl" name="map_ctrl" output="screen"/>
  <node pkg="Tus_g5" type="cur_pose" name="cur_pose" output="screen"/>
  <node pkg="Tus_g5" type="nav_control_node" name="nav_control_node" output="screen"/>
  <node pkg="Tus_g5" type="nav_node" name="nav_node" output="screen"/>
  <node pkg="Tus_g5" type="nav_error" name="nav_error" output="screen"/>
  <node pkg="Tus_g5" type="start_rviz" name="start_rviz" output="screen"/>
  <node pkg="Tus_g5" type="transform_position" name="transform_position" output="screen"/>

  <!-- Axes Velcmd -->
  <param name="axis_linear" value="1" type="int"/>
  <param name="axis_angular" value="0" type="int"/>
  <param name="scale_linear" value="1" type="double"/>
  <param name="scale_angular" value="1" type="double"/>
  <node pkg="wpb_home_bringup" type="wpb_home_js_vel" name="teleop"/>

</launch>